sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ## 3.7.5 (Unreleased)
 
+- FEATURE: Adds bulk request to container.items. Allows aggregate bulk request for up to 100 operations on items with the types: Create, Upsert, Read, Replace, Delete
+
+```js
+// up to 100 operations
+const operations: Operation[] = [
+  {
+    operationType: "Create",
+    resourceBody: { id: addEntropy("doc1"), name: "sample", key: "A" }
+  },
+  {
+    operationType: "Upsert",
+    resourceBody: { id: addEntropy("doc2"), name: "other", key: "A" }
+  },
+  {
+    operationType: "Read",
+    id: "readItemId",
+    partitionKey: `["key"]`
```

Comment:
Can we "headerify" the partition key for people? IE, let them supply `"key"`

Reply:
yeah we can